src/models/petsModel.js

```diff
@@ -151,6 +151,69 @@ const updatePetById = async (db, petID, petToUpdate, original) => {
   });
 };
 
+const getAllNews = async (db, query) => {
+  const sql = `SELECT i.*, p.*, s.ShelterName FROM PetNewsItem i
+              INNER JOIN Pet p ON i.PetID = p.PetID
```

Comment:
This is meant for a standalone news page, right? Would there be any way to just get the information from PetNewsItem here, get the Pet/Shelter information from /pets, and then combine them in the frontend? It seems like there would be less overlap that way. Let me know if I'm missing something though!

Reply:
This doesn't really matter for smaller datasets, but between doing the join in the database and combining multiple queries in the code for a larger data, my understanding is that it's faster to do it in the database, especially when it's inner join and not too complicated. Since the database is designed to handle data and queries and we can make use of indexes. I could be wrong though and I'm happy to hear different thoughts!

I thought this might be used for the home page news feed and right now I just included all related information I thought could be useful. Maybe we can table this for now and I can change it later based on the frontend needs?